src/test/java/com/dawn/jat/illuminati/post/advice/PostExceptionAdviceTest.java

```diff
@@ -16,8 +16,10 @@
 
     @Test
     void exception_throwException_returnHttpPostNotFound() {
-        PostNotFoundException postException = new PostNotFoundException("");
-        ResponseEntity<Object> rsp = advice.exception(postException);
+        PostNotFoundException postException = new PostNotFoundException("Cannot find post");
+
+        ResponseEntity<ErrorResponse> rsp = advice.exception(postException);
+        assertEquals("Cannot find post", postException.getMessage());
```

Comment:
this is not quite the necessary check, we are testing PostExceptionAdviceTest here, this one check the postException, which is an object created to fetch for advice.exception(). We should test with rsp, like the bellow line `assertEquals(HttpStatus.NOT_FOUND, rsp.getStatusCode());`

Reply:
Updated

Reply:
Updated